Content.Server/Ninja/Systems/NinjaSuitSystem.cs

```diff
@@ -21,6 +22,11 @@ public sealed class NinjaSuitSystem : SharedNinjaSuitSystem
     [Dependency] private readonly SpaceNinjaSystem _ninja = default!;
     [Dependency] private readonly PowerCellSystem _powerCell = default!;
     [Dependency] private readonly SharedTransformSystem _transform = default!;
+    [Dependency] private readonly SharedItemSystem _item = default!;
+
+    [ValidatePrototypeId<ItemSizePrototype>]
+    // The highest item size of a cell that the ninja suit should be able to contain.
+    private const string NinjaSuitInsertableCellSize = "Normal";
```

Comment:
just use the whitelist item size instead of reinventing the wheel

Reply:
Working on